beginend.el

```diff
@@ -239,6 +239,12 @@ BEGIN-BODY and END-BODY are two `progn' expressions passed to respectively
   (progn
     (prodigy-last)))
 
+(beginend-define-mode magit-status-mode
+  (progn
+    (re-search-forward "^$")
+    (forward-line))
+  (progn))
```

Comment:
I would also add some end logic, for example when you have the "Unpulled from" or "Unmerged into" headings I would like to get to the last commit (for example I want to do an interactive rebase).  Right now it goes two lines below that, which is pretty useless probably.

```
Unmerged into upstream-branch (2)
85d8881 Bar
499b347 Foo

<point ends up here>
```

Should land at the last hash.  What do you think?